Client/Get.cs

```diff
@@ -10,15 +11,53 @@ public static int File(ref FtpClient client, Commands.Get files)
             Console.WriteLine("file: " + files.File);
 
             if (files.Files.Count() > 1)
-                return MultipleFiles(files.Files);
+            {
+                Console.WriteLine($"Enter local File Path for download or press enter to skip and use current directory: ");
+                var localDir = Console.ReadLine() ?? "";
```

Comment:
Please move these lines into `MultipleFiles()`. Also, local file path should just be used by default. Specifying path overrides should be done using a command flag.

Reply:
Lines 15 and 16 above have been removed and replaced with a command property.